ddtrace/contrib/fastapi/patch.py

```diff
@@ -52,7 +53,7 @@ async def traced_serialize_response(wrapped, instance, args, kwargs):
     the result.
     """
     pin = Pin.get_from(fastapi)
-    if not pin or not pin.enabled:
+    if not pin or not pin.enabled():
```

Comment:
Crazy that this little bug has been sitting in `main` for a while!

Reply:
Credit goes to @mabdinur for finding this 😄